coordinator/src/main/scala/filodb.coordinator/queryplanner/MultiPartitionPlanner.scala

```diff
@@ -270,6 +270,15 @@ class MultiPartitionPlanner(partitionLocationProvider: PartitionLocationProvider
     (partitions, routingKeys)
   }
 
+  private def rvRangeFromLogicalPlan(logicalPlan: LogicalPlan, queryParams: PromQlQueryParams): Some[RvRange] = {
+    import LogicalPlan._
+    rvRangeFromPlan(logicalPlan) match {
+      case x @ Some(_)    =>  x
+      case None           => Some(RvRange(startMs = queryParams.startSecs * 1000L,
+        endMs = queryParams.endSecs * 1000L,
+        stepMs = queryParams.stepSecs * 1000L))
```

Comment:
Not understanding the case when we need to get start/step/end from queryParams. Can you explain the forcing factor here?

Reply:
Since other parts of the code were using ``queryParams`` I have used the same values here.